src/renderer/components/wallet/account/AccountSelector.tsx

```diff
@@ -17,7 +17,7 @@ import * as Styled from './AccountSelector.style'
 type Props = {
   selectedAsset: Asset
   walletBalances: WalletBalances
-  onChange?: (asset: Asset, walletAddress: Address) => void
+  onChange?: (asset: Asset, walletAddress: Address, network: Network) => void
```

Comment:
No need to pass `network: Network` back to `onChange` handler. Because it's already passed from parent into this component. That's `network` will be already known in parent.